docker/Dockerfile.flux

```diff
@@ -18,6 +18,10 @@ WORKDIR /home/flux
 ENTRYPOINT [ "/sbin/tini", "--", "fluxd" ]
 RUN apk add --no-cache openssh ca-certificates tini 'git>=2.3.0'
 
+# Get the kubeyaml binary (files) and put them on the path
+COPY --from=quay.io/squaremo/kubeyaml:0.1.0 /usr/lib/kubeyaml /usr/lib/kubeyaml/
```

Comment:
Hope to see 1.0 soon

Reply:
Current roadblock there: kubeyaml assumes ruamel works perfectly; however, ruamel does not work perfectly :-/

But: at least with #1094, and an annotation analogue of that, we'd catch cases where ruamel mangled things.